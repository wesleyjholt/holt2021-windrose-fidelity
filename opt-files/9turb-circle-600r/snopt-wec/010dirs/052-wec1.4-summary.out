
 ==============================
 S N O P T  7.7.3    (Dec 2019)
 ==============================
 opt-files/9turb-circle-600r/snopt-wec/010dirs/052-wec1.4-print.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-
 opt-files/9turb-circle-600r/snopt-wec/010dirs/052-wec1.4-summary.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-

 SNMEMA EXIT 100 -- finished successfully
 SNMEMA INFO 104 -- memory requirements estimated



 Nonlinear constraints      45     Linear constraints       1
 Nonlinear variables        18     Linear variables         0
 Jacobian  variables        18     Objective variables     18
 Total constraints          46     Total variables         18



 The user has defined     828   out of     828   first  derivatives


 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
     0     12               1 (1.3E-16) 1.9E-01 -4.4254019E+03     12           r
     1      1  1.0E+00      2 (5.4E-09) 1.9E-01 -4.4255206E+03     12           r
     2      6  5.5E-01      3 (4.1E-07) 5.3E-01 -4.4265483E+03      9         s  l
     3      4  1.2E-01      5  6.2E-04  4.8E-01 -4.4462246E+03     12
     4      4  9.5E-02      7  6.5E-04  2.2E-01 -4.4541208E+03     11
     5      5  1.1E-01      9  5.8E-04  4.3E-01 -4.4575884E+03     13
     6      3  9.1E-02     11  5.3E-04  3.5E-01 -4.4583336E+03     13
     7      2  1.7E-01     13  4.5E-04  4.3E-01 -4.4589398E+03     14
     8      1  2.3E-01     15  3.6E-04  3.9E-01 -4.4593175E+03     14
     9      1  2.2E-01     17  2.9E-04  3.0E-01 -4.4596199E+03     14

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    10      1  4.2E-01     19  1.7E-04  1.8E-01 -4.4597994E+03     14
    11      1  2.5E-01     21  1.3E-04  1.3E-01 -4.4599727E+03     14
    12      1  3.1E-01     23  8.9E-05  9.1E-02 -4.4600428E+03     14
    13      1  1.0E+00     24  8.9E-06  1.1E-02 -4.4601449E+03     14
    14      1  5.0E-01     26  5.6E-06  7.5E-03 -4.4601680E+03     14
    15      1  3.7E-01     28  3.9E-06  3.3E-03 -4.4601777E+03     14
    16      1  3.2E-01     30  2.7E-06  2.4E-03 -4.4601793E+03     14
    17      1  4.4E-01     32  1.5E-06  8.0E-04 -4.4601798E+03     14
    18      1  1.0E+00     33 (9.7E-09) 2.3E-04 -4.4601800E+03     14
    19      1  1.0E+00     34 (1.1E-09)(7.2E-05)-4.4601800E+03     14

 SNOPTA EXIT   0 -- finished successfully
 SNOPTA INFO   1 -- optimality conditions satisfied

 Problem name                 opt prob
 No. of iterations                  49   Objective           -4.4601799959E+03
 No. of major iterations            19   Linear    obj. term  0.0000000000E+00
 Penalty parameter           0.000E+00   Nonlinear obj. term -4.4601799959E+03
 User function calls (total)        34
 No. of superbasics                 14   No. of basic nonlinears             4
 No. of degenerate steps             0   Percentage                       0.00
 Max x                      16 9.0E+02   Max pi                     46 1.0E+00
 Max Primal infeas          56 9.8E-07   Max Dual infeas             4 7.2E-05
 Nonlinear constraint violn    9.8E-07



 Solution printed on file  18

 Time for MPS input                             0.00 seconds
 Time for solving problem                       4.25 seconds
 Time for solution output                       0.00 seconds
 Time for constraint functions                  4.25 seconds
 Time for objective function                    0.00 seconds
