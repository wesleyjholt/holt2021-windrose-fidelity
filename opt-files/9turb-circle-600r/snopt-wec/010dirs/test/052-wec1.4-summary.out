
 ==============================
 S N O P T  7.7.3    (Dec 2019)
 ==============================
 opt-files/9turb-circle-600r/snopt-wec/010dirs/052-wec1.4-print.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-
 opt-files/9turb-circle-600r/snopt-wec/010dirs/052-wec1.4-summary.out
 XXX  Keyword not recognized:         OPT-FILES/9TURB-

 SNMEMA EXIT 100 -- finished successfully
 SNMEMA INFO 104 -- memory requirements estimated



 Nonlinear constraints      45     Linear constraints       1
 Nonlinear variables        18     Linear variables         0
 Jacobian  variables        18     Objective variables     18
 Total constraints          46     Total variables         18



 The user has defined     828   out of     828   first  derivatives


 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
     0     12               1 (9.5E-07) 2.3E-01 -3.8072428E+03     12           r
     1      1  1.0E+00      2 (3.1E-08) 2.3E-01 -3.8074693E+03     12           r
     2      2  5.4E-01      3  1.7E-06  2.6E-01 -3.8091142E+03     11         s  l
     3      2  1.0E+00      4  1.8E-03  3.3E-01 -3.8188442E+03     12
     4      1  1.0E+00      5  2.2E-04  4.3E-01 -3.8235002E+03     12
     5     13  1.0E+00      7  6.7E-05  1.1E+00 -3.8306699E+03      9          m
     6     35  1.5E-03     11  4.5E-05  1.0E+00 -3.8311244E+03     12
     7      1  1.0E+00     12  5.6E-05  1.2E-01 -3.8321999E+03     12
     8      1  1.0E+00     13  1.7E-05  1.2E-01 -3.8326402E+03     12
     9      1  1.0E+00     14  2.4E-06  1.1E-01 -3.8327952E+03     12

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    10      1  1.0E+00     15  3.5E-06  1.2E-01 -3.8328742E+03     12
    11      1  5.8E-02     25  3.3E-06  1.2E-01 -3.8328805E+03     12
    12      1  2.5E-05     35  3.3E-06  1.2E-01 -3.8328805E+03     12
    13      2  4.9E-07     45  3.3E-06  9.6E-01 -3.8328805E+03     12
    14      2  1.0E+00     46  2.0E-06  1.1E-01 -3.8329159E+03     12
    15      2  1.0E+00     47 (8.9E-08) 1.2E-01 -3.8329248E+03     12
    16      2  1.0E+00     48  1.8E-06  1.2E-01 -3.8329569E+03     12
    17      2  1.0E+00     49 (1.4E-07) 1.2E-01 -3.8329738E+03     12
    18      2  1.0E+00     50 (7.4E-07) 1.2E-01 -3.8329966E+03     12
    19      2  1.0E+00     51  1.7E-06  1.1E-01 -3.8330085E+03     12

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    20      2  1.0E+00     52 (2.4E-07) 1.1E-01 -3.8330123E+03     12
    21      2  1.0E+00     53 (1.0E-07) 1.1E-01 -3.8330130E+03     12
    22      2  1.0E+00     54 (2.4E-09) 1.1E-01 -3.8330131E+03     12
    23      2  1.0E+00     55 (6.5E-09) 1.1E-01 -3.8330132E+03     12
    24      2  1.0E+00     56 (9.1E-10) 1.1E-01 -3.8330132E+03     12
    25      2  1.0E+00     57 (6.3E-11) 1.1E-01 -3.8330132E+03     12
    26      2  1.0E+00     58 (5.5E-13) 1.1E-01 -3.8330132E+03     12
    27      2  1.0E+00     59 (1.3E-13) 1.1E-01 -3.8330132E+03     12
    28      2  1.0E+00     60 (2.2E-13) 1.1E-01 -3.8330132E+03     12
    29      1  1.0E+00     61 (1.1E-13) 1.1E-01 -3.8330132E+03     12

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    30      1  1.0E+00     62 (1.4E-13) 1.1E-01 -3.8330132E+03     12
    31      1  1.0E+00     63 (2.0E-13) 1.1E-01 -3.8330132E+03     12
    32      1  1.0E+00     64 (4.3E-13) 1.1E-01 -3.8330132E+03     12
    33      1  1.0E+00     65 (9.7E-13) 1.1E-01 -3.8330132E+03     12
    34      1  1.0E+00     66 (2.4E-12) 1.1E-01 -3.8330132E+03     12
    35      1  2.7E-01     70 (2.2E-12) 1.1E-01 -3.8330132E+03     12
    36      1  9.1E-02     80 (2.3E-12) 1.1E-01 -3.8330132E+03     12
    37      1  1.8E-04     90 (2.3E-12) 1.1E-01 -3.8330132E+03     12
    38      1  3.9E-06    100 (2.3E-12) 1.1E-01 -3.8330132E+03     12
    39      1  2.1E-08    110 (2.3E-12) 1.1E-01 -3.8330132E+03     12

 Major Minors     Step   nCon Feasible  Optimal  MeritFunction     nS Penalty
    40      1  4.2E-09    120 (2.3E-12) 1.1E-01 -3.8330132E+03     12
    41      1  4.5E-10    128 (2.3E-12) 1.1E-01 -3.8330132E+03     12
    42      1  1.8E-10    135 (2.3E-12) 1.1E-01 -3.8330132E+03     12
    42      1             140 (2.3E-12) 1.1E-01 -3.8330132E+03     12           R
    43      1  8.5E-10    149 (2.3E-12) 1.1E-01 -3.8330132E+03     12         s
    43      1             155 (2.3E-12) 1.1E-01 -3.8330132E+03     12         s R
    43      1             160 (2.3E-12) 1.1E-01 -3.8330132E+03     12         s r

 SNOPTA EXIT  40 -- terminated after numerical difficulties
 SNOPTA INFO  41 -- current point cannot be improved

 Problem name                 opt prob
 No. of iterations                 122   Objective           -3.8330131990E+03
 No. of major iterations            43   Linear    obj. term  0.0000000000E+00
 Penalty parameter           0.000E+00   Nonlinear obj. term -3.8330131990E+03
 User function calls (total)       185
 No. of superbasics                 12   No. of basic nonlinears             6
 No. of degenerate steps             0   Percentage                       0.00
 Max x                      12 6.0E+02   Max pi                     39 1.0E+00
 Max Primal infeas          62 1.4E-09   Max Dual infeas             2 1.2E-01
 Nonlinear constraint violn    1.4E-09



 Solution printed on file  18

 Time for MPS input                             0.00 seconds
 Time for solving problem                      21.03 seconds
 Time for solution output                       0.02 seconds
 Time for constraint functions                 21.03 seconds
 Time for objective function                    0.00 seconds
